memory/transparent_hugepages.py

```diff
@@ -48,6 +49,9 @@ def setUp(self):
         self.mem_path = os.path.join(data_dir.get_tmp_dir(), 'thp_space')
         free_mem = int(memory.freememtotal() / 1024)
         self.dd_timeout = 900
+        self.thp_split = "thp_split_page"
+        if distro.detect().name == "rhel":
```

Comment:
pegas release still having `thp_split_page`   we need to consider that as well

Reply:
Instead of handling based on distros, handling the IndexError with either values would be my suggestion.

Reply:
@narasimhan-v As some rhel releases may have `thp_split_page`, it makes sense to do it this way, will make the changes needed.